Program.cs

```diff
@@ -16,6 +16,7 @@ static void Main(string[] args)
             PersonBookExe();
```

Comment:
You can move PrintMenu into PersonBook method. and you don't need to have initial person like huseyin here.

Reply:
done 👍